.coafile

```diff
@@ -3,3 +3,7 @@ files = **/*.py
 bears = SpaceConsistencyBear, LineLengthBear, GitCommitBear, PEP8Bear
 use_spaces = True
 max_line_length = 80
+
+[yml]
+bears = YAMLLintBears
+files = *.yml
```

Comment:
Hello @soumya-ranjan7  i think you should use `files = **.yml` instead of `files = *.yml` because `**` matches everything and `*` matches everything but directory also.

Reply:
Actually I did use **.yml before (you can see it in my previous commit) .
But on the gitter chat room I was advised to use *.yml.

Reply:
well i couldn't find your chat on gitter, did you ask reason for using `*.yml` instead this of `**.yml`,can you tell me the reason behind this because afaik it should be like that :wink:

Reply:
And as you have committed two commits ,so now you have to squash them into one.If you don't know how you squash commits , you can simply google it.

Reply:
@khanchi97 He isn't online. Tbh I have no reason as to why I should be using *.yml instead of **.yml  but I thought maybe he is right and changed it.